index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (str) => {
```

Comment:
Like Monarch Wadia mentioned in his review during twitch, maybe it would be better to have the function itself in a separate file, so you can, in the future, add more functionalities.